pgcli/packages/sqlcompletion.py

```diff
@@ -27,6 +28,15 @@ def suggest_type(full_text, text_before_cursor):
 
     identifier = None
 
+    def strip_named_query(txt):
+        pattern = re.compile(r'\\(n|nd|ns\s+[a-z]+)\s+')
```

Comment:
Sounds good. I was thinking if user started typing sql, may as well help him autocomplete it, but this is probably a better behavior. I'll change it.